MAUI/ComboBox/UI-Customization.md

```diff
@@ -698,3 +698,29 @@ Completed event can be subscribed in C# also:
 The following image illustrates the result of the above code:
 
 ![.NET MAUI ComboBox completed event](Images/UICustomization/CompletedEvent.png)
+
+## CursorPosition
+
+The cursor position in the input view can either be obtained or updated using the `CursorPosition` property in SfComboBox.
+
+{% tabs %}
+{% highlight xaml %}
+
+<editors:SfComboBox x:Name="comboBox"
+                            WidthRequest="250" 
+                            HeightRequest="35"
+                            IsEditable="True"
+                            CursorPosition = "4" />
+{% endhighlight %}
+{% highlight C# %}
+
+using Syncfusion.Maui.Inputs;
+
+SfComboBox comboBox = new SfComboBox();
+comboBox.IsEditable = true;
+comboBox.CursorPosition = 4;
+
+{% endhighlight %}
+{% endtabs %}
+
+N> Cursor position support available for editable mode only and Two way binding not supported in Android platform.
```

Comment:
N> The cursor position support is available for editable mode only, and two-way binding is not supported in the Android platform.

Reply:
Hi @Chrispineagunja,

Changed the content.